models/project-extension/src/image-regex/generator.ts

```diff
@@ -0,0 +1,24 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+
+import type { TImageRegex } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const getImageRegexOption = () => ({
+  flag: fake((f) => f.lorem.slug()),
+  replace: fake((f) => f.lorem.word()),
+  search: fake((f) => f.lorem.word()),
```

Comment:
Nit: let's try to use more real values here

Reply:
updated [97d95f9](https://github.com/commercetools/test-data/pull/562/commits/97d95f91fd46b7a4911c3a162c5a1bdbc56c649f)